src/modules/local_position_estimator/sensors/gps.cpp

```diff
@@ -189,10 +189,20 @@ void BlockLocalPositionEstimator::gpsCorrect()
 	Matrix<float, n_y_gps, n_y_gps> S = C * _P * C.transpose() + R;
 
 	// publish innovations
-	for (size_t i = 0; i < 6; i++) {
-		_pub_innov.get().vel_pos_innov[i] = r(i);
-		_pub_innov.get().vel_pos_innov_var[i] = S(i, i);
-	}
+	_pub_innov.get().gps_hvel[0] = r(3);
```

Comment:
Reorder these so that in `r` and `S` it goes in ascending order.

Reply:
Done